src/arkode/arkode_mristep.c

```diff
@@ -1007,7 +1005,7 @@ int mriStep_Init(void* arkode_mem, int init_type)
   ARKodeMem ark_mem;
   ARKodeMRIStepMem step_mem;
   int retval, j;
-  sunbooleantype reset_efun;
```

Comment:
I'd like to keep this in, since the logic below will get even more complicated when MRI adaptivity is enabled.